### DISCUSSION TITLE
All ImageSharp packages from one repo

### DISCUSSION THREAD

**Discussion by stefannikolei** [SixLabors/ImageSharp#2375] [Ideas] [answered]
Have you thought about this? 

This would probably make it easier to make changes over all Repos which have ImageSharp as foundation. 

Another thing would be streamlined releases. 
Image one uses ImageSharp and ImageSharp.Web. It will only be possible for the user to update to v3 when all packages where updated. 

What do you think, or do you have any concerns about this?

**Comment by JimBobSquarePants** [comment, marked as answer]
It makes managing version numbers a chore as they have to be synchronized. 

It also allows us to properly dogfood the public API and put serious thought into each change by forcing us to adopt each dependency in the same manner as our consumers.
